### PR TITLE
text clean up

### PR DESCRIPTION
**Pull request by dudycooly**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/idea.md [hunk 1]
```diff
@@ -7,14 +7,18 @@ assignees: ''
 
 ---
 
-## Describe Motivation
-A clear and concise description of driving factors behind this idea covering its benefits
+## Motivation
+Descirbe driving factors behind this idea covering its benefits
```
Comment (alangibson01) [inline .github/ISSUE_TEMPLATE/idea.md]:
```suggestion
Describe driving factors behind this idea covering its benefits
```

Typo

File: .github/ISSUE_TEMPLATE/idea.md [hunk 2]
```diff
@@ -7,14 +7,18 @@ assignees: ''
...
 
-### Describe Requirements
-A clear and concise description of what you want to happen.
+## Requirements
+Descirbe what you want to happen to realise the benefits listed above
```
Comment (alangibson01) [inline .github/ISSUE_TEMPLATE/idea.md]:
```suggestion
Describe what you want to happen to realise the benefits listed above
```

File: .github/ISSUE_TEMPLATE/idea.md [hunk 3]
```diff
@@ -7,14 +7,18 @@ assignees: ''
...
 
-### Describe alternatives you've considered
-A clear and concise description of any alternative solutions or features you've considered.
+## Alternatives
+Descirbe any alternative solutions or features you've considered
```
Comment (alangibson01) [inline .github/ISSUE_TEMPLATE/idea.md]:
```suggestion
Describe any alternative solutions or features you've considered
```

File: .github/ISSUE_TEMPLATE/idea.md [hunk 4]
```diff
@@ -7,14 +7,18 @@ assignees: ''
...
 
 ### Additional context
-Add any other context or screenshots about the feature request here.
+Add any other context or screenshots about the feature request here
+
+---- To be filled in by Assignee -----
+## Tasks/Feature Set
+List tasks/features in order to meet the requiremented listed above
```
Comment (alangibson01) [inline .github/ISSUE_TEMPLATE/idea.md]:
```suggestion
List tasks/features in order to meet the requirements listed above
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+3.
+
+### Workaround, if known
+List an unambiguous set of steps to unblock from this issue until a permanent fix is found
+ 
+### Additional context
+Add any other context about the problem here, how is affected you.
+
+### Your Environment
+Include as many relevant details about the environment you experienced the bug in:
```
Comment (alangibson01) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
Include as many relevant details as possible about the environment you experienced the bug in:
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+### If repeatable, how can we reproduce this issue?
+Provide a link to an example, or an unambiguous set of steps to reproduce this bug. 
+Include code to reproduce, if relevant
+1.
+2.
+3.
+
+### Workaround, if known
+List an unambiguous set of steps to unblock from this issue until a permanent fix is found
```
Comment (alangibson01) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
List an unambiguous set of steps to unblock this issue until a permanent fix is found
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+ 
+### Additional context
+Add any other context about the problem here, how is affected you.
```
Comment (alangibson01) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
Add any other context about the problem here, how it affected you.
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+labels: bug, next-bug
+assignees: ''
+
+---
+
+### What is the bug being reported?
+Describe the bug i.e. observed symptom or potential issue waiting to unfold 
+
+### What would be the expected behavior?
+Describe what should have happen to prevent the bug described above
```
Comment (alangibson01) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
Describe what should have happened to prevent the bug described above
```

**Comment by dudycooly** [comment]
> Did you mean to rename `bug_report.md` to `bug.md`? We now have 2 bug report templates

yes, unfortunately GitHub wont track that as change but create duplicate files. I will fix this

**Pull request merged**